src/iop/shadhi.c

```diff
@@ -206,7 +212,7 @@ int legacy_params(dt_iop_module_t *self, const void *const old_params, const int
     new->order = old->order;
     new->radius = old->radius;
     new->shadows = old->shadows;
-    new->reserved1 = old->reserved1;
+    new->whitepoint = old->reserved1;
```

Comment:
That doesn't feel completely safe to me...
Are you completely sure it isn't better to just verbosely default it to whatever the default/legacy-safe value?